### PR TITLE
GLPI docker image nightly build

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

Nightly publication of a docker image of bugfixes branches (9.5 only for now).

Example of `docker-compose.yml` file that can be used to run this image.
```yml
version: "3.5"

services:
    glpi:
        container_name: "glpi-9.5-bugfixes"
        image: "glpi/glpi:9.5-bugfixes"
        ports:
            - "8080:80"
        volumes:
            - "./config:/var/glpi/config"
            - "./files:/var/glpi/files"
            - "./plugins:/var/www/glpi/plugins"
    db:
        image: "mysql:5.7"
        restart: "always"
        environment:
            MYSQL_RANDOM_ROOT_PASSWORD: "yes"
            MYSQL_DATABASE: "glpi"
            MYSQL_USER: "glpi"
            MYSQL_PASSWORD: "glpi"
        volumes:
            - "./db:/var/lib/mysql"
```

Remaining tasks to do:
- [x] schedule build
- [x] configure secrets on Github Actions
- [x] clean Robofile, split tasks and make them reusable
- [ ] publish a package description on Docker Hub and Github Packages
- [x] ~logs rotation~ *log rotation has to be configured on host directly*
- [x] set installation mode to "Docker"

### CODE REVIEW

File: .docker/glpi-app/files/etc/apache2/sites-available/000-default.conf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<VirtualHost *:80>
+    DocumentRoot /var/www/glpi
+
+    ErrorLog ${APACHE_LOG_DIR}/glpi.error.log
+    CustomLog ${APACHE_LOG_DIR}/glpi.access.log combined
+
+    SetEnv GLPI_CONFIG_DIR "/var/glpi/config"
+    SetEnv GLPI_VAR_DIR "/var/glpi/files"
```
Comment (orthagh) [inline .docker/glpi-app/files/etc/apache2/sites-available/000-default.conf]: SetEnv GLPI_INSTALL_MODE "DOCKER"

File: .docker/glpi-app/files/etc/apache2/sites-available/000-default.conf [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
```
Comment (orthagh) [inline .docker/glpi-app/files/etc/apache2/sites-available/000-default.conf]:
```suggestion
SetEnv GLPI_INSTALL_MODE "DOCKER"

```

File: .docker/glpi-app/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  && docker-php-ext-install intl \
+  && apk del -f .build-deps \
+  \
+  # Install gettext required to compile locales.
+  && apk add --update gettext \
+  # Install nodejs and npm.
+  && apk add --update nodejs npm \
+  \
+  # Install git and zip used by composer when fetching dependencies.
+  && apk add --update git unzip \
```
Comment (cedric-anne) [inline .docker/glpi-app/Dockerfile]:
```suggestion
  && apk add --update git unzip zip \
```

**Comment by cedric-anne** [comment]
As discussed offline, Docker image build will be done using another repository.

**Pull request closed without merging**